src/utils/orgs.ts

```diff
@@ -29,3 +29,9 @@ export const getPackageDirs = async (): Promise<string[]> => {
 const getConfigValue = <T extends AnyJson>(key: string): T => {
   return ConfigAggregator.getValue(key)?.value as T;
 };
+
+export const getSourceApiVersion = async (): Promise<string> => {
+  const project = await SfdxProject.resolve();
+  const projectConfig = await project.resolveProjectConfig();
+  return getString(projectConfig, 'sourceApiVersion');
```

Comment:
```suggestion
  return projectConfig?.sourceApiVersion;
```

Reply:
@mdonnalley should this default to target org instanceApiVersion?